src/modules/hunting/cves.py

```diff
@@ -61,6 +62,16 @@ def __init__(self, binary_version):
         self.evidence = "kubectl version: {}".format(self.binary_version)
 
 
+""" kube-state-metrics CVE """
+
+class KubeStateMetricsVulnerability(Vulnerability, Event):
+    """"""
+    def __init__(self, version):
+        Vulnerability.__init__(self, KubectlClient, "kube-state-metrics Vulnerable To CVE-2019-17110", category=InformationDisclosure, vid="KVH051")
```

Comment:
I would categorize as `AccessRisk`, same as other secret leaking vulnerabilities detected by kube-hunter

Reply:
Makes sense, I will change it. Thanks :)